hms-api/serverless.yml

```diff
@@ -2,17 +2,36 @@ service: hms-api
 
 plugins:
   - serverless-plugin-typescript
-  - serverless-localstack
+  - serverless-dynamodb-local
+  - serverless-offline
 
 custom:
-  localstack:
-    debug: true
-    stages: [ local ]
-    autostart: true
-    lambda:
-      mountCode: false
-    docker:
-      sudo: false
+  dynamodb:
+    stages:
+      - local
+    start:
+      port: 8000
+      inMemory: true
```

Comment:
having this set to false will allow you to restart with the same data, might be nice :thinking:

Reply:
but then `dbPath` should also be specified maybe :thinking: